### PR TITLE
NEW: Separate read-only and adding time on projects

### PR DESCRIPTION
**Pull request by FHenry**
Issue : #18162

### CODE REVIEW

**Comment by eldy** [comment]
Can you provide an example of use case for this need ?

**Comment by FHenry** [comment]
@eldy Input spend time should be not the same as read task time. External user, for exemple, should be able see task time, but input spent time should different.

**Comment by FHenry** [comment]
@eldy Could you have a look ?

File: htdocs/core/modules/modProjet.class.php [hunk 1]
```diff
@@ -202,6 +202,12 @@ public function __construct($db)
 		$this->rights[$r][4] = 'all';
 		$this->rights[$r][5] = 'supprimer';
 
+		$r++;
+		$this->rights[$r][0] = 145; // id de la permission
+		$this->rights[$r][1] = "Can enter time consumed on assigned tasks (timesheet)"; // libelle de la permission
```
Comment (eldy) [inline htdocs/core/modules/modProjet.class.php:207]: So the label of permission 41 and permission 142 should be modified. They already bring permission to enter time.
If you move this into a dedicated permission, you must update any use + any label of permission 41 but also 142

**Pull request merged**